store/copr/batch_coprocessor.go

```diff
@@ -410,6 +410,15 @@ func (b *batchCopIterator) handleBatchCopResponse(bo *tikv.Backoffer, response *
 		return errors.Trace(err)
 	}
 
+	if len(response.RetryRegions) > 0 {
+		for _, retry := range response.RetryRegions {
+			id := tikv.NewRegionVerID(retry.Id, retry.RegionEpoch.ConfVer, retry.RegionEpoch.Version)
+			logutil.BgLogger().Info("invalid region because tiflash detected stale region", zap.String("region id", id.String()))
```

Comment:
Will this statement print many logs when a tiflash peer is down?

Reply:
How about print the `len(retryRegions)` and three of them.

Reply:
fixed, only print the first ten ones.